Can't build tutorial from the website

I'm at the very first step of the [website's tutorial](https://docs.libp2p.io/tutorials/getting-started/go/) and end up getting an error when trying to build:
```
# github.com/libp2p/go-libp2p/p2p/host/basic
../../../../pkg/mod/github.com/libp2p/go-libp2p@v6.0.23+incompatible/p2p/host/basic/basic_host.go:131:32: cannot use h (type *BasicHost) as type host.Host in argument to identify.NewIDService:
        *BasicHost does not implement host.Host (missing EventBus method)
../../../../pkg/mod/github.com/libp2p/go-libp2p@v6.0.23+incompatible/p2p/host/basic/basic_host.go:158:32: cannot use h (type *BasicHost) as type host.Host in argument to ping.NewPingService:
        *BasicHost does not implement host.Host (missing EventBus method)
../../../../pkg/mod/github.com/libp2p/go-libp2p@v6.0.23+incompatible/p2p/host/basic/basic_host.go:512:28: h.natmgr.NAT().ExternalAddrs undefined (type *"github.com/libp2p/go-libp2p-nat".NAT has no field or method ExternalAddrs)
../../../../pkg/mod/github.com/libp2p/go-libp2p@v6.0.23+incompatible/p2p/host/basic/natmgr.go:94:8: assignment mismatch: 1 variable but "github.com/libp2p/go-libp2p-nat".DiscoverNAT returns 2 values
../../../../pkg/mod/github.com/libp2p/go-libp2p@v6.0.23+incompatible/p2p/host/basic/natmgr.go:94:26: not enough arguments in call to "github.com/libp2p/go-libp2p-nat".DiscoverNAT
        have ()
        want (context.Context)
../../../../pkg/mod/github.com/libp2p/go-libp2p@v6.0.23+incompatible/p2p/host/basic/natmgr.go:154:13: mapping.InternalAddr undefined (type "github.com/libp2p/go-libp2p-nat".Mapping has no field or method InternalAddr)
../../../../pkg/mod/github.com/libp2p/go-libp2p@v6.0.23+incompatible/p2p/host/basic/natmgr.go:176:26: not enough arguments in call to nat.NewMapping
        have (multiaddr.Multiaddr)
        want (string, int)
../../../../pkg/mod/github.com/libp2p/go-libp2p@v6.0.23+incompatible/p2p/host/basic/natmgr.go:206:13: mapping.InternalAddr undefined (type "github.com/libp2p/go-libp2p-nat".Mapping has no field or method InternalAddr)
```
I'm using go 1.13.

--- a/README.md
+++ b/README.md
@@ -63,7 +63,13 @@ You can start using go-libp2p in your Go application simply by adding imports fr
 import "github.com/libp2p/go-libp2p"
 ```
 
-The next time you run `go get` or `go build`, the Go build tools will look for [available releases](https://github.com/libp2p/go-libp2p/releases), and will pick the highest available one.
+Run `go get` or `go build`, excluding the libp2p repos from Go modules proxy usage. You only need to do this the first time you import go-libp2p to make sure you latch onto the correct version lineage (see [golang/go#34189](https://github.com/golang/go/issues/34189) for context):
+
+```sh
+$ GOPRIVATE='github.com/libp2p/*' go get ./...
+```
+
+The Go build tools will look for [available releases](https://github.com/libp2p/go-libp2p/releases), and will pick the highest available one.
 
 As new releases of go-libp2p are made available, you can upgrade your application by manually editing your `go.mod` file, or using the [Go tools](https://golang.org/cmd/go/#hdr-Maintaining_module_requirements) to maintain module requirements.
 

